lib/src/navbar/navbar-menu/navbar-menu.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit, Input, Inject, OnDestroy } from '@angular/core';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+import { IconModule } from '../../icon/icon.module';
+import { IAppSwitcherService, IDiscoveryApplication } from '../../app-switcher/app-switcher-interfaces';
+import { Subscription } from 'rxjs/Subscription';
+import { Subject } from 'rxjs/Subject';
+
+@Component({
+    selector: 'hc-navbar-menu',
+    templateUrl: './navbar-menu.component.html',
+    styleUrls: ['./navbar-menu.component.scss'],
+    animations: [
```

Comment:
This uses web animations which has limited support in browsers currently
https://caniuse.com/#feat=web-animation
There are three options:
1) no animations for un supported browsers (but they would still be 100% functional)
2) add shim and instructions on library consumers
3) convert it to use css transitions

Reply:
It looks great and I am impressed. I just ran into these limitations...

Reply:
My fault @benjanderson - you mentioned that a couple days ago and then I promptly ignored it.  Thanks for the reminder, and fortunately the css transition seems to work just as well as the web animation, and uses less code too.  Interestingly the easing is a little bit different between the two, but it's barely enough to notice.